qemu/tests/dump_guest_memory.py

```diff
@@ -0,0 +1,202 @@
+import logging
+import operator
+import os
+
+from avocado.utils import process
+
+from virttest import utils_misc
+from virttest import utils_package
+
+
+def run(test, params, env):
+    """
+    Test dump-guest-memory, this case will:
+
+    1) Start VM with qmp enable.
+    2) Check if host kernel are same with guest
+    3) Connect to qmp port then run qmp_capabilities command.
+    4) Initiate the qmp command defined in config (qmp_cmd)
+    5) Verify that qmp command works as designed.
+    6) Verify dump file with crash
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+
+    def check_env(session):
+        """
+        Check if host kernel version is same with guest
+
+        :param session: vm session
+        :type session: aexpect.client.ShellSession
+        """
+        host_version = process.run("uname -r")
+        guest_version = session.cmd("uname -r")
+        if host_version != guest_version:
```

Comment:
`process.run("uname -r")` returns `avocado.utils.process.CmdResult` while `session.cmd("uname -r")`  returns only the output from the command prompt, so here I think the comparison is not correct.
And again, you query 'uname -r' from host twice, that's not necessary I think.

Reply:
Ok,updated.Thanks!